resolwe_bio/processes/support_processors/multiqc.py

```diff
@@ -0,0 +1,299 @@
+"""Prepare MultiQC report."""
+import json
+import os
+from plumbum import TEE
+
+from resolwe.process import (
+    BooleanField,
+    Cmd,
+    DataField,
+    DirField,
+    FileHtmlField,
+    GroupField,
+    IntegerField,
+    ListField,
+    Process,
+    StringField
+)
+
+
+def create_symlink(src, dst):
+    """Create a symbolic link."""
+    return Cmd['ln']('-s', '--backup=numbered', src, dst)
+
+
+def create_summary_table(samples, species, build):
+    """Prepare sample summary MultiQC table."""
+    sample_summary = {
+        'id': 'sample_summary',
+        'section_name': 'Sample Summary',
+        'plot_type': 'table',
+        'file_format': 'json',
+        'data': {}
+    }
+
+    for sample in zip(samples, species, build):
```

Comment:
It also works if you expand the variables directly in for loop (see comment below)